src/core/Signature.ts

```diff
@@ -74,4 +81,67 @@ export class Signature implements ISignature {
     this.redirectUrlTTL = params.redirectUrlTTL;
     this.state = params.state;
   }
+
+  /**
+   * Create a new signature instance
+   * @param requestBuilder Service account request builder
+   * @param body POST signature request body
+   * See more info here: https://stg-docs.int.algoan.net/api/#operation/createSignatureDocument
+   */
+  public static async create(
+    requestBuilder: RequestBuilder,
+    folderId: string,
+    body: PostSignatureDTO,
+  ): Promise<Signature> {
+    const signature: ISignature = await requestBuilder.request({
+      url: `/v1/folders/${folderId}/signatures`,
+      method: 'POST',
+      data: body,
+    });
+
+    return new Signature(folderId, signature, requestBuilder);
+  }
+
+  /**
+   * Get a signature instance from a folder by id
+   * @param folderId Unique folder identifier
+   * @param signatureId Unique signature identifier
+   */
+  public static async getById(
+    requestBuilder: RequestBuilder,
+    folderId: string,
+    signatureId: string,
```

Comment:
If forgot a remark.

Maybe it is better to pass these two arguments as object. It is what I did in my side because I think it is less error prone than passing two arguments of the same type (if the order is false, the request is wrong).

What do you think?

Reply:
I think you"re right, I'm correcting this